gr-blocks/python/blocks/qa_file_source_sink.py

```diff
@@ -104,6 +105,67 @@ def test_file_source_can_seek_after_open(self):
 
         os.remove(filename)
 
+    def test_begin_tag(self):
+        src_data = range(1000)
+        expected_result = range(1000)
+
+        filename = "tmp.32f"
```

Comment:
You can use Python's tempfile module and ``with`` construct here, which will create a unique filename and remove it automatically.